core/cap-0067.md

```diff
@@ -275,6 +275,14 @@ Where `from` is the account paying the fee, either the fee bump fee account or t
 An additional `fee` event will be emitted if there is a refund, where the amount is a negative value, indicating that `from`'s balance will be increased by the amount. This second `fee` event will only appear if a refund occurs, so classic transactions and Soroban transactions that do not have a refund will only emit one `fee` event.
 The final fee paid can be calculated by taking the sum of the amounts from all `fee` events.
 
+#### Consuming fee events in chronological order
+
+While the fee events appear within `TransactionMetaV4`, consuming those events in chronological order will require some additional logic - 
```

Comment:
I think it's worth noting that:

1. currently, the only possible events that can be found at the transaction level (`TransactionMetaV4.events`) are fee events.
2. The list of events in `TransactionMetaV4.events` must always have a length of 1 or 2.
3. The chronological order of operation events is obtained by enumerating through the transactions in apply order and, for each transaction, enumerating the events in `TransactionMetaV4. OperationMetaV2.

I think providing all those details fully specifies the ordering of all events found in the ledger